salt/tests/unit/modules/test_cordon.py

```diff
@@ -0,0 +1,175 @@
+from parameterized import parameterized
+
+import salt
+from salt.exceptions import CommandExecutionError
+
+from salttesting.mixins import LoaderModuleMockMixin
+from salttesting.unit import TestCase
+from salttesting.mock import MagicMock, patch
+
+import metalk8s_cordon
+
+
+class CordonTestCase(TestCase, LoaderModuleMockMixin):
+    """
+    TestCase for `cordon` module
+    """
+    loader_module = metalk8s_cordon
+
+    def node_object(self, unschedulable=None, with_taints=False):
+        node_obj_dict = {
+            'api_version': 'v1',
+            'kind': 'Node',
+            'metadata': {
+                'name': 'bootstrap'
+            },
+            'spec': {
+                'taints': [{
+                    'effect': 'NoSchedule',
+                    'key': 'node-role.kubernetes.io/bootstrap'
+                }],
+                'unschedulable': unschedulable
+            }
+        }
+        if with_taints:
+            node_obj_dict['spec']['taints'].append({
+                'effect':
+                'NoSchedule',
+                'key':
+                'node.kubernetes.io/unschedulable'
+            })
+        return node_obj_dict
+
+    def test_virtual(self):
+        """
+        Tests the return of `__virtual__` function
+        """
+        self.assertEqual(metalk8s_cordon.__virtual__(), 'metalk8s_kubernetes')
+
+    def test_check_deps_success(self):
+        """
+        Tests the availability of `metalk8s_kubernetes.update_object` module
+        in salt
+        """
+        with patch.dict(metalk8s_cordon.__salt__, {
+                'metalk8s_kubernetes.update_object': MagicMock()
+        }):
+            metalk8s_cordon._check_deps()
+
+    def test_check_deps_fail(self):
+        """
+        Tests the unavailability of `metalk8s_kubernetes.update_object` module
+        in salt
+        """
+        self.assertRaisesRegexp(
+            CommandExecutionError,
+            "'metalk8s_kubernetes.update_object' is not available",
+            metalk8s_cordon._check_deps)
+
+    @parameterized.expand([
+        (None,),
+        (True,),
+    ])
+    def test_cordon_node(self, unschedulable, **kwargs):
```

Comment:
kwargs is not needed here and add one with False

```suggestion
    @parameterized.expand([
        (None,),
        (True,),
        (False,)
    ])
    def test_cordon_node(self, unschedulable):
```